interface/wx/stdpaths.h

```diff
@@ -408,10 +408,14 @@ class wxStandardPaths
     virtual wxString GetUserLocalDataDir() const;
 
     /**
-        Return OS specific directory where project shared liraries are
+        Return OS specific directory where project shared liraries are.
+        The function is different from GetPluginsDir() as PlugIns
+        and Shared Libraries are 2 different things (especially on Mac).
+        Plugin is a library that can be used without restartng the applicatioon
+        whereas the library needes to be installed and the application restarted.
 
         - Windows: @c it returns the folder where the application binary is located
-        - Unix:    @c it returns the libraries installation path
+        - Unix:    @c it returns the libraries installation path, i.e. /usr/lib
         - Mac:     @c "/Applications/exename.app/Contents/Frameworks/"
```

Comment:
This is wrong now and should be
```suggestion
        - Mac:     @c "/Applications/exename.app/Contents/Frameworks"
```

Reply:
@vadz ,
Fixed in the upcoming commit.